docs/how-to-guides/javascript/README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with JavaScript
+# Getting Started with JavaScript and how it is used with WordPress
```

Comment:
```suggestion
# Getting Started with JavaScript and how it is used with Gutenberg
```

Reply:
sounds good; or 'the block editor' (instead of Gutenberg... not sure which is preferred nowadays).